gui/widgets/roiview.cpp

```diff
@@ -122,3 +132,12 @@ void BoundedRect::mouseMoveEvent(QGraphicsSceneMouseEvent *ev)
 	// remember where we took off
 	lastcursor = cursor;
 }
+
+void BoundedRect::mousePressEvent(QGraphicsSceneMouseEvent *ev)
+{
+	if (ev->buttons() & Qt::RightButton) {
+		QGraphicsRectItem::mousePressEvent(ev);
+	}
+	lastcursor = QPointF(0.f, 0.f);
+	setCursor(Qt::SizeAllCursor);
```

Comment:
This doesn't seem right. SizeAllCursor should only be set when the mouse is pressed for movement. Here it is set after the menu closes. I guess we need if () {} else {} here.